main.py

```diff
@@ -35,7 +41,17 @@ def main() -> int:
 
         print("Drone's destination is not final waypoint.")
 
-        time.sleep(DELAY_TIME)
+        time.sleep(DELAY_TIME_1)
+
+    camera = CameraDevice(0, 100, LOG_NAME)
```

Comment:
Does this run? `LOG_NAME` is of type `Path` but the constructor expects a string.